src/theory/sets/theory_sets_private.cpp

```diff
@@ -922,12 +894,6 @@ void TheorySetsPrivate::postCheck(Theory::Effort level)
   Trace("sets-check") << "Sets finish Check effort " << level << std::endl;
 }
 
-bool TheorySetsPrivate::preNotifyFact(TNode atom, bool polarity, TNode fact)
```

Comment:
`preNotifyFact` is still declared in `theory_sets_private.h`. It should be removed if no longer needed.

Reply:
Good catch, will remove.